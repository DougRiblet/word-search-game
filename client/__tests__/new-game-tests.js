import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import NewButton from '../components/New-Game/new-game-present';

describe('New Game', () => {
  test('Snapshot', () => {
    const NewButton_tree = renderer.create(
      <NewButton newGame={() => newGame()} allownew={true} />
    ).toJSON();
    expect(NewButton_tree).toMatchSnapshot();
  });

  test('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<NewButton newGame={() => newGame()} allownew={true} />, div);
  });
});
