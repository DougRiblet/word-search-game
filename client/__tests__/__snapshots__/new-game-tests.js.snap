// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`New Game Snapshot 1`] = `
<button
  disabled={false}
  id="new-game-button"
  onClick={[Function]}
>
  New Game
</button>
`;
